package ndnfs;

message FileInfo
{
  // File attributes from Qiuhan's earlier implementation
  required int32 size = 1;
  required int32 totalseg = 2;
  required int32 version = 3;
  
  // Mime type is available in name branch <file>/_meta/mime_type, and sent along with other attributes
  optional string mimetype = 4;
}

